master/static/migrations/20220311095512_add_workspace_tables.up.sql

```diff
@@ -0,0 +1,25 @@
+CREATE TABLE workspaces (
+  id SERIAL PRIMARY KEY,
+  name TEXT UNIQUE NOT NULL,
```

Comment:
**note:** Looking at some of the other migration sql, for fields like this, it seems like the common pattern they use is...
```
name character varying UNIQUE NOT NULL
```
I think they might be the [same under the hood](https://stackoverflow.com/a/4849030/5402432), so implementation-wise it doesn't matter, but might be worth consideration if we want to keep them consistent.

Reply:
personally i prefer text unless you specifically need varchar(n)/char(n).

Reply:
I will keep it as text, but thanks for checking on this. We don't have any set reason to limit name length with varchar type.